### ISSUE TITLE
Cannot produce a working windows binary from template project with no changes

### ISSUE THREAD

**Issue by ofersadgat** [electron-userland/electron-builder#8232]
electronuserland/builder:wine-mono (latest) hash: 04ebd993c08b

v18.18.2 (from the docker image)

30.0.7
* **Electron Type (current, beta, nightly)**:
current

win32

### Operating system

Windows 10 (Version 10.0.19045 Build 19045)

### Expected behavior

Electron binary to run

### Actual behavior

Electron binary doesnt run and instead produces the error (when run from a command line): 

[293384:0523/095224.285:FATAL:archive_win.cc(152)] Failed to find file integrity info for resources\app.asar
[293384:0523/095224.285:ERROR:crashpad_client_win.cc(868)] not connected

### Steps to reproduce

I use WSL for an Ubuntu 20.04 command line, but it shouldnt make a difference (since I build in a docker container):

1. npm init electron-app@latest my-app -- --template=webpack-typescript
2. cd my-app
3. docker run -it -v ${HOME}:${HOME} -v /etc/passwd:/etc/passwd -u `id -u`:`id -g` -v ${PWD}:/repo electronuserland/builder:wine-mono /bin/bash
4. cd /repo
5. yarn install
6. yarn run package --platform win32 (this succeeds without error)

To see error:
7. In windows, open file explorer to the 'out' directory
8. Double click electron-typescript.exe (nothing happens)
9. Open the same directory in powershell
10. ./electron-typescript.exe
11. Observe error

### Additional information

Note: when I use a native environment (e.g. powershell + windows node) the same steps above do work. They just dont work while running the docker image (which is what I would like to use for CI). I have also tried running the docker image from powershell and it produces the same result.

**Comment by mook-as** [comment]
If you're using [ASAR integrity](https://www.electronjs.org/docs/latest/tutorial/asar-integrity) on macOS, it's now supported on Windows as of Electron 30.  It doesn't look like electron-builder supports that yet (see 8245), so disabling [the fuse](https://www.electronjs.org/docs/latest/tutorial/fuses#embeddedasarintegrityvalidation) on Windows might help.

**Comment by ofersadgat** [comment]
Yup, that does it. I'd rather not disable the fuse, so I'll wait for the PR to merge. Thanks!
